pkg/skaffold/sync/sync_test.go

```diff
@@ -758,7 +758,9 @@ func TestNewSyncItem(t *testing.T) {
 			})
 
 			actual, err := NewItem(ctx, test.artifact, test.evt, test.builds, &mockConfig{}, 0)
-
+			if test.expected != nil {
```

Comment:
Normally we change each test data (I know it may be more work) in lieu of adding additional logic in the unittest, so as we don't need a unittest for unittests. 🤷